src/main/java/fi/metatavu/dcfb/server/search/SearchConsts.java

```diff
@@ -0,0 +1,12 @@
+package fi.metatavu.dcfb.server.search;
+
+/**
+ * Constrants for search operations
+ */
+public class SearchConsts {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=3&issues=AWQWmPQFbXXTsx7O6YI8&open=AWQWmPQFbXXTsx7O6YI8)

[ISSUE_KEY]: AWQWmPQFbXXTsx7O6YI8